docker_registry_client/manifest.py

```diff
@@ -0,0 +1,108 @@
+# code extracted from https://git.io/vM0EB used under license:
+#
+# Copyright (c) 2015 David Halls <https://github.com/davedoesdev/>
```

Comment:
@davedoesdev I'd like to make this a lot smaller, so with your permission can I change this to be just one line.

```python
# code extracted from https://git.io/vM0EB used under license (MIT).
```